azkaban-web-server/src/main/java/cloudflow/services/ExecutionServiceImpl.java

```diff
@@ -192,10 +198,74 @@ public JobExecution getJobExecution(String executionId, String jobDefinitionId,
       attempts.add(new JobExecutionAttempt(id, startTime, endTime, status));
     }
 
-    JobExecutionAttempt lastAttempt =
-        new JobExecutionAttempt(attempts.size(), jobNode.getStartTime(), jobNode.getEndTime(),
-            jobNode.getStatus());
+    JobExecutionAttempt lastAttempt = new JobExecutionAttempt(attempts.size(),
+        jobNode.getStartTime(), jobNode.getEndTime(), jobNode.getStatus());
     attempts.add(lastAttempt);
     return attempts;
   }
+
+  @Override
+  public String createExecution(ExecutionParameters executionParameters) throws CloudFlowException {
+    requireNonNull(executionParameters);
+    // TODO ypadron: check user permissions
+
+    // TODO ypadron: obtain flow and project dynamically
+    Project project = this.projectManager.getProject("yb-test");
+    Flow flow = project.getFlow("flowPriority2");
```

Comment:
Why are these values hard-coded? Is that what the `todo` is for ?

Reply:
yes, I need to come back here and fix it once flow endpoints are ready.

Reply:
It should be possible to use the `ProjectManager` to get this information, could you please check ?

Reply:
The id of flows table today is the name of that flow. To create an execution in CF we're given a "real" flow id but plumbing for that is not yet implemented so there is no method in ProjectManager that fetches a flow by id.

Reply:
I was referring to using something like `flow = project.getFlow(flowId)` until the flow endpoint has progressed further (this code will likely not change once that happens as well). I am ok delaying this to a subsequent PR.

Reply:
The problem is that the project object is not available without the flow object in this endpoint (we only receive a flow id). I should be getting the project from the flow, not the other way around, that's why ProjectManager alone doesn't help here.
I prefer to wait until flow changes are ready to implement this part properly.